### PR TITLE
remove counter

### PR DESCRIPTION
**Pull request by vancefarren**
The original LightDB demonstrated several different features in one sample.  The presence of the counter in the LED example is an artifact of this previous intent.  The counter is unrelated to the LED handling and is represented in a separate sample.

<a href="https://gitpod.io/#https://github.com/golioth/zephyr-sdk/pull/120"></a>

### CODE REVIEW

**Comment by 0Grit** [comment]
I recall this discussion from our review. Can you summarize the context of it in a sentence or two in the PR description?

**Review by mniestroj — CHANGES_REQUESTED** [review]
> I recall this discussion from our review. Can you summarize the context of it in a sentence or two in the PR description?

The same should be done for commit message. There needs to be a reason for each change and this reason should always be mentioned in commit message. Having full context of each change inside git allows us to be independent of Github and allows to easily search through history of the repo, without entering Github UI.

See also: https://github.com/golioth/zephyr-sdk/blob/main/CONTRIBUTING.md#style--commit-guide

Commit title should be something like: `samples: lightdb_led: remove counter`. If in doubt, just look into previous commit messages in similar area / directory, e.g. with `git log --oneline -- samples/lightdb_led`.

The exact reason why `gitlint` job is failing right now is to prevent commiting patches that do not have proper commit body, i.e. description of why this change is desirable.

**Comment by 0Grit** [comment]
> 
> See also: https://github.com/golioth/zephyr-sdk/blob/main/CONTRIBUTING.md#style--commit-guide
> 
> Commit title should be something like: `samples: lightdb_led: remove counter`. If in doubt, just look into previous commit messages in similar area / directory, e.g. with `git log --oneline -- samples/lightdb_led`.

MCU developers and hardware engineers like me will struggle with this. I want to maintain the rigor of Zephyr/Linux but will keep an eye out for a middle ground. 

Or at least look for ways to ease their/our transition into the civilized world.

**Comment by vitprajzler** [comment]
I agree with Marcin's reasoning. Having the right level of context within the git scope (rather than being dependent on 3rd party tool) prevents problems when performing / reviewing git actions in the future.

It is something to get used to work-flow wise. But since we aim to contribute to Zephyr itself, the effort is well worth it. And our own repo is a good starting point for that (lower stakes than Zephyr contributions).

**Comment by 0Grit** [comment]
@vancefarren Do you need to update any of the user facing docs as a result of this PR?

**Review by 0Grit — APPROVED** [review]

**Comment by vancefarren** [comment]
> @vancefarren Do you need to update any of the user facing docs as a result of this PR?

I will look into it

**Comment by mniestroj** [comment]
Commit author:
```
Author:     vancefarren <vancefarren@gmail.com>
```
Commit body:
```
...
Signed-off-by: Vance Farren <vance@golioth.io>

Co-authored-by: Vance Farren <vance@golioth.io>
```

Really? :)

**Comment by 0Grit** [comment]
Could have swore we unified these before force push. Guess we got none of them.

> Commit author:
> 
> ```
> Author:     vancefarren <vancefarren@gmail.com>
> ```
> 
> Commit body:
> 
> ```
> ...
> Signed-off-by: Vance Farren <vance@golioth.io>
> 
> Co-authored-by: Vance Farren <vance@golioth.io>
> ```
> 
> Really? :)
